server/prisma/migrations/20240131062714_update_order_process/migration.sql

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
Just a note but seeing as migrations are really useful for environments where we care about keeping the contents of the database, it might make sense to at some point, delete all the previous migrations, and start fresh on the migrations seeing as they are building up and don't add much value at the moment.

Reply:
I agree, this seems like a good opportunity to start fresh with a new schema, ergo delete all the migrations and recreate the deployed DBs. We'd have to coordinate with others working on schema changes though

Reply:
That would also make this a little better to review

Reply:
Fixed